answerking_app/utils/schema/schema_examples.py

```diff
@@ -118,7 +118,7 @@
 problem_detail_example: ProblemDetails = {
     "errors": {"name": "The name field is required."},
     "type": "https://testserver/problems/error/",
-    "title": "One or more validation errors occurred.",
+    "title": "One or more permissions errors occurred.",
```

Comment:
probably would change this to "One or more validation or permissions errors occurred."